### DISCUSSION TITLE
黑暗大帝贝利亚已经入侵绳网了！快点投降吧！

### DISCUSSION THREAD

**Discussion by Qiling-XwX** [share121/inter-knot#1024] [其他]
就此投降！没有人会受伤
![0880FDC6AF0C4204DC92668E894D7C3D](https://github.com/user-attachments/assets/de40e636-0152-4b2d-ae5b-a960cf88b3af)

**Comment by Furina-D** [comment]
麻麻我害怕QAQ

**Comment by 2Mnn** [comment]
光之战士会拯救我们的！贝利亚，你早晚得投降！

**Comment by su2569** [comment]
流萤救我[doge]
